source/administration/release-lifecycle.rst

```diff
@@ -101,3 +101,25 @@ During each monthly release, Mattermost backports high severity or high impact s
 +-------------+-----------------------+--------------------------+--------------------------+--------------------------+
 | 4.0         | Feature               |  2017-07-16              | 2017-09-15               |                          |
 +-------------+-----------------------+--------------------------+--------------------------+--------------------------+
+
+Desktop App Release Lifecycle
+------------------------------------
+There are no hard requirements for Self-Managed/Desktop App compatibility, but certain Self-Managed features that involve the operating system require a specific Desktop App version. Security and high severity bug fixes are normally not backported to previous Desktop App versions. Please review the `Desktop App changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_ notes for any specific Self-Managed or Desktop App requirements for features and functionalities, as well as notes on security fixes. The Desktop App doesn't have a specific release cadence, but we follow the same release cycle pattern for all upcoming releases as shown below.
+
++-------------+-----------------------+--------------------------+
+| Version     | Lifecyle Start Date   | Lifecycle End Date       |
++=============+=======================+==========================+
+| 4.6         | 2020-10-16            | TBD                      |
++-------------+-----------------------+--------------------------+
+| 4.5         | 2020-06-16            | 2020-10-15               |
```

Comment:
I can start a google doc. I thought that there was a technical limitations for why we don't backport high severity fixes for desktop and mobile apps, but sounds like it's just a process thing that can be improved.

Reply:
most of the time is because electron's release cycle is a bit faster than ours. e.g.: v4.6.2 uses electron 7 which right now is not supported anymore by electron. This would mean that we can provide support for our own code but we might not be able to provide security backports for the libs that we rely on.